core-web/libs/block-editor/src/lib/plugins/dot-bubble-menu.plugin.ts

```diff
@@ -308,7 +308,12 @@ export class DotBubbleMenuPluginView extends BubbleMenuView {
     }
 
     setChangeToOptions() {
-        const changeToOptions = suggestionOptions.filter((item) => item.id != 'horizontalLine');
+        const allowedBlocks: string[] = this.editor.storage.dotConfig.allowedBlocks;
+
+        const changeToOptions =
+            allowedBlocks.length > 1
+                ? suggestionOptions.filter((item) => allowedBlocks.includes(item.id))
+                : suggestionOptions.filter((item) => item.id != 'horizontalLine');
```

Comment:
What happens if I set a block in the white list and is misspelled?

Reply:
What about passing this from the init? I think that is better is we can pass the block from the init and plugins and extension are only in charge of render what they get, a more centralized way.

Reply:
if is misspelled we ignore it. About the centralized way,  the "change" to will have a limited blocks, and will not be the same a suggestions. for that propuse we hace a "core" list `suggestionOptions`